pages/doc.js

```diff
@@ -233,6 +235,9 @@ export default class Documentation extends Component {
   }
 }
 
+/**
+ * Container styled <div> component
+ */
 const Container = styled.div`
```

Comment:
These I now realize may be too much to document each, but what do you guys think?

Reply:
def no comments like this :)

Reply:
Agree. Removed all the styled component doc comments.